src/components/UploadSection.svelte

```diff
@@ -0,0 +1,80 @@
+<script lang="ts">
+    import {onMount} from 'svelte';
+    import { getUploadRequest } from '../lib/server/ApiCLient';
+    import { type UploadRequest } from '../lib/server/ApiCLient';
+  
+    let  files = [];
+    let uploadedFiles: UploadRequest[];
+
+    onMount(async()=>{
+        try{
+        uploadedFiles  = await getUploadRequest();
+        if(uploadedFiles === null){
+            throw new Error;
+        }
+        }
+        catch(e){
+            console.error(e);
+            uploadedFiles = null;
+        }
```

Comment:
1.  Please enable auto formatting in your IDE
2. What is the point of throwing an error and then catching it ?